### PR TITLE
model: add fritzbox 4040 dsa model for snapshot

### PR DESCRIPTION
**Pull request by PolynomialDivision**

### CODE REVIEW

File: group_vars/model_avm_fritzbox_4040_dsa.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+dsa_ports:
+  - lan1
+  - lan2
+  - lan3
+  - lan4
+  - wan
+
+wireless_devices:
+  - name: 11a_standard
+    hwmode: 11a
```
Comment (Noki) [inline group_vars/model_avm_fritzbox_4040_dsa.yml:18]: hwmode is deprecated. please use band instead. please also set htmode_prefix.

File: group_vars/model_avm_fritzbox_4040_dsa.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    path: platform/soc/a800000.wifi
+    ifname_hint: wlan5
+  - name: 11g_standard
+    hwmode: 11g
```
Comment (Noki) [inline group_vars/model_avm_fritzbox_4040_dsa.yml:22]: same as above

**Review by Noki — CHANGES_REQUESTED** [review]

**Comment by spolack** [comment]
```
Error: _vars/model_avm_fritzbox_4040_dsa.yml:1:1: [error] missing document start "---" (document-start)
```

Please fix that lint error, then feel free to merge

**Comment by Noki** [comment]
htmode_prefix is still missing

**Comment by spolack** [comment]
You are right, sorry!

**Review by Noki — APPROVED** [review]

**Pull request merged**